addons/io/org.openhab.io.openhabcloud/src/main/java/org/openhab/io/openhabcloud/internal/CloudClient.java

```diff
@@ -427,7 +427,7 @@ public void sendNotification(String userId, String message, String icon, String
                 notificationMessage.put("severity", severity);
                 socket.emit("notification", notificationMessage);
             } catch (JSONException e) {
-                logger.error(e.getMessage());
+                logger.error("{}", e.getMessage());
```

Comment:
I would log the exception instead, it is an error, I want to be able to debug the stacktrace